packages/@o3r/logger/src/services/logger/logger.service.ts

```diff
@@ -50,17 +57,37 @@ export class LoggerService implements Logger {
     return sessionUrls.length <= 1 ? sessionUrls[0] : sessionUrls;
   }
 
+  /**
+   * Register a new client to the logger service
+   *
+   * @param client
+   */
+  public registerClient(client: LoggerClient) {
```

Comment:
maybe you should expose the a getter of the client list as well?

Reply:
what for?

Reply:
List the registered clients to not register several time the same and wait for the warning saying it has been registered several time?